docs/source/usecases/VirtualFunctions.md

```diff
@@ -111,6 +111,61 @@ deviceInstance->setAField(someHostValue); // set some field on the host
 
 This is the solution that the code teams we have talked to have said is the most productive way to solve the problem.
 
+## But what if I do not really need the V-Tables on the device side?
+
+The performance critical part of a code might not use pointers to base class and dynamic polymorphism to compute a result from data.
+Thus, the device might not need to have a working virtual function table.
+Consider the following example:
```

Comment:
Only beef I have what is being added is how it is introduced.  The current version reads somewhat confusing, I had to step back and remember what all this was about before reading on.